config.plist/security.md

```diff
@@ -0,0 +1,42 @@
+# Apple Secure Boot
+
+These settings in your config.plist can restrict which macOS versions OpenCore will boot. You will want to check these really quick before booting your USB.
+
+## Misc
+
+### Security -> SecureBootModel
+
+OpenCore by default has [Apple Secure Boot](https://dortania.github.io/OpenCore-Post-Install/universal/security/applesecureboot.html#what-is-apple-secure-boot) enabled.
+This enables security features such as verification of macOS' `boot.efi`, though also has the side effect of restricting which macOS versions OpenCore will boot.
```

Comment:
Should explain that if you want a combo of ie. High Sierra and Big Sur, you need Disabled (if your SMBIOS is not T2 and newer than the minimum version for that Mac)